mcp/1.0/modules/ROOT/pages/mcp-connector-studio.adoc

```diff
@@ -0,0 +1,126 @@
+
+= Using Anypoint Studio to Configure {connector-name} Connector {version} - Mule 4
+:exchange-search-name: mcp
+:connector-name: MCP
+:version: 0.1.0-BETA
+:lc-connector-name: mcp
+:studio-connector-name: MCP Connector
+:tls:
+
+include::connectors::partial$t-studio-partial.adoc[tags=topic-intro]
+
+
+[[create-mule-project]]
+== Create a Mule Project
+
+include::connectors::partial$t-studio-partial.adoc[tags=create-project-intro]
+
+[[add-connector-to-project]]
+== Add the Connector to Your Mule Project
+
+include::connectors::partial$t-studio-partial.adoc[tags=add-connector-to-project]
+
+[[configure-source]]
+== Configure a Source
+
+include::connectors::partial$t-studio-partial.adoc[tags=configure-source-intro]
+You can configure one of these sources to use with {connector-name} Connector:
+
+////
+If the connector has connector-specific sources, list them first, using one bullet for each source. If the connector has only one source, do not use a list.
+////
+
+
+* *<connector-specific input source>*
++
+<description>
++
+include::connectors::partial$t-studio-partial.adoc[tags=listener-scheduler-descriptions]
+
+
+////
+Use one source as an example. If the connector has a connector-specific source, use one of those sources as an example. In the example, list the required and important fields.  If the connector does not have a connector-specific source, use *HTTP > Listener*, using the partial shown below.
+////
+
+include::connectors::partial$t-studio-partial.adoc[tags=listener-steps]
+
+[[add-connector-operation]]
+== Add a Connector Operation to the Flow
+
+include::connectors::partial$t-studio-partial.adoc[tags=add-connector-operation]
+
+[[configure-global-element]]
+== Configure a Global Element for the Connector
+
+include::connectors::partial$t-studio-partial.adoc[tags=global-element-intro]
+
+include::connectors::partial$t-studio-partial.adoc[tags=property-placeholders]
+
+include::connectors::partial$t-studio-partial.adoc[tags=global-element-first-steps]
+
+. In the *General* tab, in *Connection*, select the authentication method to configure. The MCP protocol's security is mostly based on the OAuth Authorization Code flow. To keep it compatible with different systems and uses, the connector also supports other authentication methods, including:
+* <link to authentication method config section>
```

Comment:
i know you said you're working on the auth stuff, but may want to add links/anchors to the existing topics here

* <<basic_authentication>> 
...etc

Reply:
Yes, thanks!